experimental/traffic-portal/src/app/shared/navigation/navigation.service.ts

```diff
@@ -186,6 +186,10 @@ export class NavigationService {
 				{
 					href: "/core/iso-gen",
 					name: "Generate System ISO"
+				},
+				{
+					href: "/core/certs/ssl",
+					name: "Inspect Cert"
```

Comment:
I  think this should spell out "Certificate" - I'd even vote to be more specific: "SSL Certificate"